module/VuFind/src/VuFind/Auth/ShibbolethWithWAYF.php

```diff
@@ -41,6 +42,12 @@
  */
 class ShibbolethWithWAYF extends AbstractShibboleth
 {
+
+    /**
+     * Configured IdPs with entityId and overridden attribute mapping
+     *
+     * @param \Laminas\Config\Config
```

Comment:
I think the Travis build is complaining because this should be `@var` rather than `@param`.